doc/changelog/10-coqide/18524-coqide.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Loading dropped files whose name contains special characters
+  (`#17421 <https://github.com/coq/coq/pull/17421>`_,
+  fixes `#3977 <https://github.com/coq/coq/issues/3977>`_,
```

Comment:
I had no idea that CoqIDE could do drag and drop.  That would be a good thing to mention in the CoqIDE section of the documentation (one sentence should do it, eg "You can open files by dragging their icons from your file manager and dropping them in the CoqIDE window.")